web-registry/src/components/organisms/DetailsSection/DetailsSection.tsx

```diff
@@ -0,0 +1,143 @@
+import { Grid, useMediaQuery, useTheme } from '@mui/material';
+
+import { BlockContent } from 'web-components/lib/components/block-content';
+import { CredibilityCard } from 'web-components/lib/components/cards/CredibilityCard/CredibilityCard';
+import Section from 'web-components/lib/components/section';
+import ResponsiveSlider from 'web-components/lib/components/sliders/ResponsiveSlider';
+import { Body, Label, Title } from 'web-components/lib/components/typography';
+import { headerFontFamily, Theme } from 'web-components/lib/theme/muiTheme';
+
+import { DetailsSectionButton } from './DetailsSection.Button';
+import {
+  CREDIT,
+  VIEW_CREDIT_CLASS_DOC,
+  VIEW_METHODOLOGY,
+} from './DetailsSection.constants';
+import { DetailsSectionCredit } from './DetailsSection.Credit';
+import { useSectionStyles } from './DetailsSection.styles';
+import { DetailsSectionProps } from './DetailsSection.types';
+
+export const DetailsSection: React.FC<
+  React.PropsWithChildren<DetailsSectionProps>
+> = ({
+  header,
+  credibilityCards,
+  methodology,
+  creditClassDoc,
```

Comment:
pending answer from Sam with regards to the missing credit class doc field in the metadata

Reply:
cc @S4mmyb